README.md

```diff
@@ -106,8 +106,7 @@ context propagation.
 
 Apache 2.0 - See [LICENSE][license-url] for more information.
 
-[ci-image]: https://circleci.com/gh/open-telemetry/opentelemetry-ruby.svg?style=svg
-[ci-url]: https://circleci.com/gh/open-telemetry/opentelemetry-ruby
+[ci-image]: https://github.com/open-telemetry/opentelemetry-ruby/workflows/CI/badge.svg
```

Comment:
I suggest appending `?event=push` so the badge reflects the latest continuous (push to master) CI rather than the latest PR presubmit run.